lib/vehicle.rb

```diff
@@ -6,13 +6,17 @@ class Vehicle
               :make,
               :model,
               :engine
+  attr_accessor :plate_type, :registration_date, :registered
```

Comment:
Good call here on making these items accessor's since you'll need to change them later.